src/test/java/qna/domain/AnswerTest.java

```diff
@@ -22,22 +24,42 @@
     @Autowired
     UserRepository userRepository;
 
+    public User javajigi;
+    public User sanjigi;
+
+    public Question questionJavajigi;
+
+    @BeforeEach
+    void setUp() {
+        javajigi = userRepository.save(UserTest.JAVAJIGI);
+        sanjigi = userRepository.save(UserTest.SANJIGI);
+        questionJavajigi = questionRepository.save(new Question("title1", "contents1").writeBy(javajigi));
+    }
+
     @Test
     void save() {
-        final Long id = answerRepository.save(A1).getId();
-        assertThat(id).isEqualTo(1);
+        final Answer answer = answerRepository.save(new Answer(sanjigi, questionJavajigi, "Answers Contents1"));
```

Comment:
new 생성자를 통해 모든 테스트에서 중복적으로 객체 생성을 하지 않고 `테스트 픽스처` 클래스를 만들어서 사용해보는것도 좋을 것 같네요 :)

```suggestion
        final Answer answer = answerRepository.save(AnswerFixture.답변());
```

위에는 예시고, 실제 픽스처의 이름은 해당 픽스처의 의도를 명확히 드러내주는게 좋을 것 같습니다.
`AnswerFixture.ID가_없는_답변` 이런식으로 드러내도 좋을 것 같아요~